### PR TITLE
Merge master into dev-2020.4.

### PR DESCRIPTION
**Pull request by jsun28**
Merge master into dev-2020.4.

### CODE REVIEW

**Comment by lukewrites** [comment]
@rosswbrown I had the same question. `master` is going into `dev-2020.4` at d79b853 but there's lots of other stuff, too.

@jsun28 can you confirm?

File: README.md [hunk 1]
```diff
@@ -4,10 +4,11 @@
 
 This project consists of documentation, example files, the Tableau Datasource Verification Tool (TDVT) test harness, and a packaging tool that you can use to build and customize a Tableau Connector that uses an ODBC or JDBC driver.
 
-The latest version of the SDK is always targeted towards the latest, non-beta version of Tableau. Right now, this is **Tableau Desktop/Server 2020.2**. This means that the samples may not work on older versions of Tableau, and connectors packaged with newer versions of the SDK may not work in older versions of Tableau.
+The latest version of the SDK is always targeted towards the latest, non-beta version of Tableau. Right now, this is **Tableau Desktop/Server 2020.3**. This means that the samples may not work on older versions of Tableau, and connectors packaged with newer versions of the SDK may not work in older versions of Tableau. You can download past releases of the SDK to work with older versions of Tableau if necessary.
```
Comment (lukewrites) [inline README.md:7]: Might want to link to the Releases page for this.

**Comment by jsun28** [comment]
> d79b853

**Comment by jsun28** [comment]
close this, will do a PR for merging master into dev-2020.4 directly

**Pull request closed without merging**